### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by okigan**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -4,6 +4,7 @@
   "description": "Integration between React Router and React-Bootstrap",
   "main": "./lib/index.js",
   "scripts": {
+    "update-dep": "./node_modules/.bin/npm-check-updates -u",
```
Comment (jquense) [inline package.json]: the `./node_modules/.bin/` is unecessary anything in that directory is automatically in the PATH when running an npm script so `.npm-check-updates -u` is enough

**Comment by taion** [comment]
Did you even look at what this did before you just ran `ncu` (which should be installed globally anyway)? Like... just no, this breaks the entire damn package because of stuff like Babel 5 -> Babel 6.

**Comment by okigan** [comment]
Yes, I saw that change, and as per your CONTRIBUTING ran the  `npm run visual-test` which looked the same -- hence the pull request. Would you mind to expand the test or better yet add a travis.ci config to auto check pull requests. 

Other than that, what's the plan to update history dependency as it's it's breaking `npm shrinkwrap`?

**Comment by jquense** [comment]
in what way do our dependencies break shrinkwrap? npm doesn't care about having a bunch of dirrent, or out of date deps.

**Comment by taion** [comment]
There was a Travis CI config and it failed your PR. There's actual tests that, if you ran, would also have failed.

**Pull request closed without merging**